services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Bearing.java

```diff
@@ -3,12 +3,14 @@
 import androidx.annotation.NonNull;
 import com.google.auto.value.AutoValue;
 
+import java.util.Map;
+
 /**
  * A value pair of an {@link #angle} which is a clockwise value from true north between 0 and 360,
  * and a {@link #degrees()} which is the range of degrees by which the angle can deviate.
  */
 @AutoValue
-public abstract class Bearing {
+public abstract class Bearing extends DirectionsJsonObject {
```

Comment:
Some objects didn't extend `DirectionsJsonObject`. Now all models extend it

Reply:
we're potentially expect that any object can have unrecognized fields, let's add `abstract Builder unrecognised(Map<String, Object> value);` to `DirectionsJsonObject`

Reply:
added, thank you for the suggestion